js/src/widgets/metadataView.js

```diff
@@ -164,11 +163,7 @@
     var label = attr;
 
     label = label.replace(/^@/, '');
-    label = label.replace(/([A-Z])/g, ' $1');
     label = label.replace(/\s{2,}/g, ' ');
-    label = label.replace(/\w\S*/g, function(txt) {
```

Comment:
The problem is that while the CSS can be modified by MIrador users, the capitalization via JavaScript is currently hard-coded. It would be good to offer this option, since the convention of title-casing every word is pretty much an English-only thing, or at least it is not commonly used in German. Maybe we could restrict this style to English labels and labels that do not declare a language?